tools/test.py

```diff
@@ -1381,7 +1383,7 @@ def BuildOptions():
         dest="suppress_dialogs", default=True, action="store_true")
   result.add_option("--no-suppress-dialogs", help="Display Windows dialogs for crashing tests",
         dest="suppress_dialogs", action="store_false")
-  result.add_option("--shell", help="Path to V8 shell", default="shell")
+  result.add_option("--shell", help="Path to Node executable", default=None)
```

Comment:
This should either be `node executable` (i.e. the name of the Node.js binary) or `Node.js executable`.